src/components/Table/Table.jsx

```diff
@@ -52,8 +52,8 @@ const sortCustomersByName = (customers, sortOrder) => {
 }
 
 useEffect(() => {
-    setCustomersToRender(sortCustomersByName(customersData,sortByName));
-}, [sortByName])
+    setSortedCustomers(sortCustomersByName(customersData,sortByName));
+}, [sortByName,customersData])
```

Comment:
This is a good usage of `useEffect`.

Sort only when `customersData` or `sortByName` changes

Reply:
Ok